From 987a04e2157a755ec47cafe89cb31e5928f43415 Mon Sep 17 00:00:00 2001
From: Arun P Madhavan <arun_madhavan@comcast.com>
Date: Wed, 19 Jan 2022 20:49:07 +0000
Subject: [PATCH 1/2] COMCAST-794: RDK Voice stack - T4H support

Reason for change: Temporary - Add T4H Notus S1 Voice keycode support
for controlMgr. Remove once T4H RCU Firmware adopts RDK RCU Spec.
Test Procedure: Enable respective feature and test key trigger.
Risks: Low
License: Inherited
Upstream-Status: Pending
Source: RDKM

Signed-off-by: Arun Madhavan <arun_madhavan@comcast.com>
---
 src/ble/ctrlm_hal_ble.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/ble/ctrlm_hal_ble.cpp b/src/ble/ctrlm_hal_ble.cpp
index 60e5f1c..5c218c5 100644
--- a/src/ble/ctrlm_hal_ble.cpp
+++ b/src/ble/ctrlm_hal_ble.cpp
@@ -63,7 +63,7 @@ using namespace std;
 #define CTRLM_HAL_BLE_SKY_RCU_DBUS_INTERFACE_PROPERTIES     "org.freedesktop.DBus.Properties"
 #define CTRLM_HAL_BLE_SKY_RCU_DBUS_GET_PROPERTIES           "org.freedesktop.DBus.Properties.Get"
 
-#define CODE_VOICE_KEY      66
+#define CODE_VOICE_KEY      217
 #define KEY_INPUT_DEVICE_FILE_IR     "/dev/input/event0"
 #define KEY_INPUT_DEVICE_BASE_DIR    "/dev/input/"
 #define KEY_INPUT_DEVICE_BASE_FILE   "event"
-- 
2.17.1

